include/openenclave/bits/seal.h

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+/**
+ * @file seal.h
+ *
+ * This file defines constants and structures for sealing APIs.
+ *
+ * Only TEE agnostic definitions should go to this file. SGX specific
+ * definitions should go to sgx/seal.h
+ */
+
+#ifndef _OE_BITS_SEAL_H
+#define _OE_BITS_SEAL_H
+
+#include <openenclave/bits/evidence.h>
+#include <openenclave/bits/types.h>
+
+OE_EXTERNC_BEGIN
+
+typedef struct _oe_seal_setting_header
+{
+    int name;
```

Comment:
```suggestion
    int type;
```
"name" implies a string, which this is not.  Recommend "type" which is commonly used in TLV structs.  Or if you don't like that, "id" is also used elsewhere in OE.

Reply:
Ok. I'll change `name` to `type`, `size` to `length` and the actual setting to `value`, to follow the *TLV* convention.